theory/material/linear-elastic.md

```diff
@@ -0,0 +1,51 @@
+# Linear Elastic
+
+The linear elastic model is the simpliest model for homogeneous isotropic materials. Relevant equations are presented briefly below.
+
+## Elastic Modulus
+
+The bulk, shear, and constrained moduli ($K$, $G$, and $M$, respectively) are defined as follows, 
+
+$$ K = \frac{E}{3 (1 - 2\nu)}, $$
+
+$$ G = \frac{E}{2(1 + \nu)}, $$
+
+$$ M = \frac{E(1 - \nu)}{(1 + \nu)(1 - 2 \nu)} , $$
+
+where $E$ is the Young's modulus and $\nu$ is the Poisson's ratio.
+
+## Wave Velocities
+
+The shear and pressue wave velocities ($v_s$ and $v_p$, respectively) are defined [1] as follows,
+
+$$ v_s = \sqrt{\frac{M}{\rho}}, $$
+
+$$ v_p = \sqrt{\frac{G}{\rho}}, $$
+
+where $\rho$ is the density.
+
```

Comment:
Could you please add that these information are automatically computed and is available as a JSON material property and can be accessed as: `property["pwave_velocity"]` and `property["swave_velocity"]`. I think that would be useful.

Reply:
Automatic computation and storage as JSON material properties added in newest commit